scripts/KATCalPipe.py

```diff
@@ -32,14 +32,15 @@
 									'This option is only used when --polcal is selected.')
 parser.add_option("--katdal_timeout", default=300, type='int', help='Timeout (in seconds) for data read in katdal')
 parser.add_option("--katdal_retries", default=2, type='int', help='Number of retries for data read in katdal')
+parser.add_option("--antphase-adjust-filename", default=None, help="string or None\nfilename to numpy .npz file containing phase adjustment per input per channel\nto be applied to all raw visibilities before further processing.")
 (options, katfilenames) = parser.parse_args()
 
 if len(katfilenames) == 0:
     parser.print_help()
     sys.exit()
 
 kwargs = {}
-for k in ['parmFile', 'scratchdir', 'targets', 'configFile', 'timeav', 'flag', 'reuse', 'zapraw', 'aipsdisk', 'halfstokes', 'gzip', 'dropants', 'blmask', 'refant', 'katdal_refant', 'polcal', 'XYtarg', 'delaycal_mvf', 'katdal_timeout', 'katdal_retries']:
+for k in ['parmFile', 'scratchdir', 'targets', 'configFile', 'timeav', 'flag', 'reuse', 'zapraw', 'aipsdisk', 'halfstokes', 'gzip', 'dropants', 'blmask', 'refant', 'katdal_refant', 'polcal', 'XYtarg', 'delaycal_mvf', 'katdal_timeout', 'katdal_retries', 'antphase_adjust_filename']:
```

Comment:
I think we should add it as a parameter file argument rather than a command line argument.

You just need to add a new parameter to the dictionary defined in https://github.com/mauch/KATObitPipe/blob/master/katim/KATCal.py#L33 

It will be picked up and the default overridden when someone supplies a parameter file.